packages/shader-shaderlab/src/shaders/shadingPBR/FragmentPBR.glsl

```diff
@@ -302,10 +311,24 @@ SurfaceData getSurfaceData(Varyings v, vec2 aoUV, bool isFrontFacing){
     #endif
 
     #ifdef MATERIAL_ENABLE_TRANSMISSION 
-            surfaceData.transmission = material_Transmission;
-            #ifdef MATERIAL_HAS_TRANSMISSION_TEXTURE
-                surfaceData.transmission *= texture2D(material_TransmissionTexture, uv).r;
+        surfaceData.transmission = material_Transmission;
+        #ifdef MATERIAL_HAS_TRANSMISSION_TEXTURE
+            surfaceData.transmission *= texture2D(material_TransmissionTexture, uv).r;
+        #endif
+
+        surfaceData.absorptionCoefficient = -log(material_AttenuationColor + HALF_EPS) / max(HALF_EPS, material_AttenuationDistance);
+
+        #ifdef MATERIAL_HAS_THICKNESS
+            #if REFRACTION_MODE == 2
+                //https://zh.wikipedia.org/wiki/%E8%96%84%E8%86%9C%E5%85%89%E5%AD%A6
+                surfaceData.thickness = 0.005;
```

Comment:
`material_Thickness` unit is world space,1 world space is 1meter, 500 nm=0.0000005m